geopandas/tools/_random.py

```diff
@@ -0,0 +1,150 @@
+import numpy
+from ..array import points_from_xy
+from ..geoseries import GeoSeries
+from ..geodataframe import GeoDataFrame
+import pygeos
```

Comment:
Note that pygeos needs to be an optional dependency. If possible, the method should work without it or raise an informative error (this would also fail on CI).